lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1454,6 +1454,22 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value mult = b.create<arith::MulFOp>(loc, self, alpha);
     return b.create<arith::SubFOp>(loc, other, mult);
   }
+  if (isa<AtenToDtypeOp>(op)) {
+    Value input = payloadArgs[0];
+    Type inType = input.getType();
+    Type outType = payloadArgs[1].getType();
+    Value result;
+    if (inType.isF32()) {
+      if (inType == outType)
+        result = input;
+      else if (outType.isInteger(64))
+        result = b.create<arith::FPToSIOp>(loc, b.getI64Type(), input);
+      else if (outType.isInteger(1))
+        result = b.create<arith::FPToSIOp>(loc, b.getI1Type(), input);
```

Comment:
Need an error msg for unhandled case here. something like `atenToDtype.emitError("unimplemented: unsupported target dtype");`